### ISSUE TITLE
shorthand_operator false positive

### ISSUE THREAD

**Issue by keith** [realm/SwiftLint#1211]
```swift
var foo = [3, 4]
var bar = [1, 2]
foo = bar + foo
```

This generates a warning to use `+=` instead, which doesn't work here because of the ordering.

**Comment by keith** [comment]
@marcelofabri you might be interested in this one.

**Comment by marcelofabri** [comment]
Fixed in #1212.

**Comment by keith** [comment]
Thanks!
